turbopy/diagnostics.py

```diff
@@ -85,6 +85,9 @@ class PointDiagnostic(Diagnostic):
         Function for assigned output method: standard output or csv.
     csv : :class:`numpy.ndarray`, None
         numpy.ndarray being written as a csv file.
+    interval : int, None
```

Comment:
I think we should use a more specific name, since `dump_interval` is already used e.g. in the `FieldDiagnostic`. How about `write_interval`?